lib/Communications.rb

```diff
@@ -296,6 +303,18 @@ def self.default_logger(level) # :nodoc:
       logger.formatter = proc { |severity, datetime, progname, msg| "#{msg}\n" }
       logger
     end
+
+    def self.set_request_headers_host(req)
+      req_headers = req.instance_variable_get('@header')
```

Comment:
Haven't seen this pattern before, is this `@header` part of the standard ruby Request lib?